utils/prof_utils.py

```diff
@@ -228,7 +227,7 @@ def sigmaClip(data, alpha=3., tol=0.1, ntrials=10):
         oldstd = newstd
 
 #---------------------------------------------------------------
-def check_clip(clipped_prof, toomuch=0.8, toolittle=0.):
+def check_clip(clipped_prof, toomuch=0.8, toolittle_frac=0., toolittle_absolute=4):
```

Comment:
Update the docstring to explain the new option

Reply:
Done